cve_bin_tool/cli.py

```diff
@@ -589,13 +587,24 @@ def main(argv=None):
     if int(args["cvss"]) > 0:
         score = int(args["cvss"])
 
+    metrics = args["metrics"]
     epss_percentile = 0
-    if float(args["epss_percentile"]) > 0 or float(args["epss_percentile"]) < 100:
+    if (
+        args["epss_percentile"]
+        and float(args["epss_percentile"]) >= 0
+        and float(args["epss_percentile"]) <= 100
+    ):
+        metrics = True
         epss_percentile = float(args["epss_percentile"]) / 100
         LOGGER.debug(f"epss percentile stored {epss_percentile}")
 
     epss_probability = 0
```

Comment:
We should probably log a warning if the epss_percentile or epss_probability was invalid before setting it to 0, but that's not your mistake.